### DISCUSSION TITLE
Rider Plugin gone?

### DISCUSSION THREAD

**Discussion by yankun** [nuke-build/nuke#1269] [Q&A]
Hi, maybe a stupid question, but I'm unable to find an up to date rider plugin for nuke. The one I have is outdated and is not compatible with the most recent stable release. I've searched the market place, but nothing with nuke can be found:

https://plugins.jetbrains.com/search?search=NUKE

Where has the plugin gone? Thanks!

**Comment by matkoch** [comment]
Subscribe here for updates: https://github.com/nuke-build/nuke/issues/1262
